lms/djangoapps/onboarding/forms.py

```diff
@@ -262,9 +358,81 @@ def save(self, request, commit=True):
 
         selected_function_areas = get_actual_field_names(request.POST.getlist('function_areas'))
         user_info_survey.user.extended_profile.save_user_function_areas(selected_function_areas)
+
+        is_currently_unemployed = self.cleaned_data['is_currently_employed']
+
+        user = user_info_survey.user
+        first_name = user.first_name
+        last_name = user.last_name
+
+        if not is_currently_unemployed:
+            organization_name = self.cleaned_data.get('organization_name', '').strip()
+            is_poc = self.cleaned_data['is_poc']
+            org_admin_email = self.cleaned_data['org_admin_email']
+
+        if not is_currently_unemployed and organization_name:
+            organization_to_assign = Organization.objects.filter(label__iexact=organization_name).first()
```

Comment:
Let's please use get or create.

Reply:
It's old code... so just leaving it for now.